lib/action-creators.js

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
Sorry for being a bit picky, can we have a tiny bit of dock describing what this file does?